### PR TITLE
Make sure that block inlining doesn't cause validation to fail

### PR DESCRIPTION
**Pull request by Kroppeb**

### CODE REVIEW

**Review by SuperCoder7979 — CHANGES_REQUESTED** [review]
Hey, I just have one minor nitpick before merging :)

File: src/org/jetbrains/java/decompiler/modules/decompiler/InlineSingleBlockHelper.java [hunk 1]
```diff
@@ -99,6 +100,28 @@ private static void inlineBlock(SequenceStatement seq, int index) {
...
+          StatEdge labelEdge = iterator.next();
+          if (labelEdge.getType() == StatEdge.TYPE_BREAK &&
+              MergeHelper.isDirectPath(last, labelEdge.getDestination())) {
+            // find the correct closure
+            Statement closure = parent;
+            while(!MergeHelper.isDirectPath(closure, labelEdge.getDestination())) {
+              closure = closure.getParent();
+              if (closure == null) {
+                // euh, where have we been inlined to?
+                throw new RuntimeException("Where have we been inlined to?");
```
[the remark this answers is not in the document]
Comment (Kroppeb) [reply src/org/jetbrains/java/decompiler/modules/decompiler/InlineSingleBlockHelper.java:117]: The comment doesn't feel entirely correct. It's more like "no valid closure".

[the remark this answers is not in the document]
Comment (SuperCoder7979) [reply src/org/jetbrains/java/decompiler/modules/decompiler/InlineSingleBlockHelper.java]: The idea was to make the comment more presentable, at least :P

**Review by SuperCoder7979 — APPROVED** [review]
Good now, thanks!

**Pull request merged**